lms/templates/fields/field_readonly.underscore

```diff
@@ -1,10 +1,8 @@
 <label class="u-field-title" aria-hidden="true">
```

Comment:
@cptvitamin Thanks, I didn't notice this label. First of all, we don't need to render it if there is no title (which is the case here). But when there is a title, I don't know why it was a label (since it was never linked to the input field it was describing).

I'm going to try changing the label to a span when it is actually needed, and not render it otherwise.  I assume the aria-hidden is here because of the explicit sr span on line 4.